guides/common/modules/proc_viewing-multiple-content-view-environment-hosts.adoc

```diff
@@ -0,0 +1,47 @@
+[id="Viewing_multiple_content_view_environment_hosts{context}"]
+= Viewing multiple content view environment hosts
+
+Use this procedure to view multiple content view environment activation keys in Hammer CLI
+
+.Prerequisites
+Register a host with the created activation key. 
+For more information, see xref:Using_Activation_Keys_for_Host_Registration_{context}[Using activation keys for host registration].
+
+[NOTE]
+====
+Click the *Insecure* checkbox when registering a host.
+====
+
+.Procedure
+. In the {ProjectwebUI}, navigate to *Hosts* > *All Hosts*.
+. Click the *New UI* button.
+. Click *Manage columns*.
+. Under *Content*, click the *Content view environments* checkbox and click *Save*.
+. The multiple content view environments for the host are displayed under the *Content view environments* column.
```

Comment:
The column displays information for both single-environment and multi-environment hosts.

```suggestion
. Content view environments for each host are displayed under the *Content view environments* column.
```